### PR TITLE
[Enhancement - UI] Exercise-update components checkbox size adjustment

### PR DESCRIPTION
**Pull request by Olimbia23**
### Checklist
- [X] Client: I added multiple screenshots/screencasts of my UI changes

### Motivation and Context
Checkboxes (Options) had different size. This PR unifies the size and adjusts them properly.

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration (course should have presentation score enabled)
3. Navigate to Exercises
4. Create/Update an exercise
5. Look at checkboxes

#### Before:
Screenshot 2021-08-07 at 19 00 43

#### After:
Screenshot 2021-08-07 at 18 45 43

### CODE REVIEW

**Review by ndricimrr — APPROVED** [review]
Tested on TS1. Size changes look good 👍

**Comment by julian-christl** [comment]
All files importing `_exercise-update.scss`, e.g. `text-exercise-update.scss` contain the following scss:
```css
label {
    font-size: 17px;
    font-weight: bolder;
}
```
which matches your scss apart from the identifier. Could this be refactored in a way?

File: src/main/webapp/app/exercises/shared/presentation-score/presentation-score.component.ts [hunk 1]
```diff
@@ -29,6 +29,14 @@ import { Authority } from 'app/shared/constants/authority.constants';
...
         </ng-container>
     `,
+    styles: [
+        `
+            .form-check-label {
+                font-size: 15px;
+                font-weight: bolder;
+            }
+        `,
+    ],
```
Comment (julian-christl) [inline src/main/webapp/app/exercises/shared/presentation-score/presentation-score.component.ts:39]: Could we use a `styleUrls` here instead? Seems like an unnecessary duplication here to me.

**Review by Malyuk-A — DISMISSED** [review]
Tested on TS1. Looks good.

**Review by SimonEntholzer — DISMISSED** [review]
Looks good. One thing I don't really get though is why you need the form-check-label class twice. Do different checkboxes use different classes?

File: src/main/webapp/app/exercises/shared/presentation-score/presentation-score.component.ts (hunk 1 above)
Comment (iozbeyli) [reply src/main/webapp/app/exercises/shared/presentation-score/presentation-score.component.ts:39]: I agree doing it in two places will create the problem again in the future.

**Comment by Olimbia23** [comment]
> All files importing `_exercise-update.scss`, e.g. `text-exercise-update.scss` contain the following scss:
> 
> ```css
> label {
>     font-size: 17px;
>     font-weight: bolder;
> }
> ```
> 
> which matches your scss apart from the identifier. Could this be refactored in a way?

Hey @julian-christl. Thank you for your review. I applied your suggestion 😊

File: src/main/webapp/app/exercises/text/manage/text-exercise/text-exercise-update.scss [hunk 2]
```diff
@@ -1,6 +1 @@
 @import '../../../shared/exercise/_exercise-update.scss';
-
```
Comment (julian-christl) [inline src/main/webapp/app/exercises/text/manage/text-exercise/text-exercise-update.scss:2]: I think you can remove this file and use the _exercise-update.scss instead

File: src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.scss [hunk 3]
```diff
@@ -1,6 +1 @@
 @import '../../shared/exercise/_exercise-update.scss';
-
```
Comment (julian-christl) [inline src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.scss:2]: I think you can remove this file and use the _exercise-update.scss instead

File: src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.scss [hunk 4]
```diff
@@ -1,6 +1 @@
 @import '../../shared/exercise/_exercise-update.scss';
-
```
Comment (julian-christl) [inline src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.scss:2]: I think you can remove this file and use the _exercise-update.scss instead

**Review by julian-christl — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.scss (hunk 4 above)
Comment (Olimbia23) [reply src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.scss:2]: good idea. I removed all of them. @julian-christl

**Review by julian-christl — APPROVED** [review]
Good job, Code looks good.

**Review by MerlinMehmed — APPROVED** [review]
Gode looks good and works as expected on all exercise types

**Review by Malyuk-A — APPROVED** [review]
Tested on TS3. Code looks good. Re-approve. 👍

Concerning the current client test fail:
Other devs and I have similar issues.
One way to fix it seems to add tests to the components you touched. (Could be a small unrelated test to a not yet fully tested functionality to increase the code/test coverage)
image

**Review by lschlesinger — APPROVED** [review]
I tested on TS1, works as described, the checkboxes have the same size now.

**Review by ndricimrr — APPROVED** [review]
Re-Approve. Tested again on all types and works well.

**Review by argertboja — APPROVED** [review]
Tested on TS1, LGTM 👍🏼

**Review by jpbernius — APPROVED** [review]

**Pull request merged**